rust/tools/authorization-logic/src/lib.rs

```diff
@@ -41,7 +41,12 @@ pub extern "C" fn generate_datalog_facts_from_authorization_logic(
     let decl_skip_vec = unsafe {
         CStr::from_ptr(c_decl_skip)
     }.to_str().unwrap().split(',').map(|s| s.to_string()).collect();
-    compilation_top_level::compile(input_filename,
+    // The CLI interface has been extended to take multiple files as inputs, but the C library does 
+    // not make use of this because it is not needed, yet. Doing so would just entail changing this 
+    // interface and the code that consumes it; the internals of the auth logic compiler should not 
+    // need to change -- the following line can just call compile(...)
+    // instead of compile_one_prgram(...)
+    compilation_top_level::compile_one_program(input_filename,
```

Comment:
Is this information even important? Is, from the C program's perspective, compiling a multiprogram any different from preprocessing a multiprogram to be a single program?

Also, line wrapping.

Reply:
I left this comment because it could be useful to have this library handle combining multiple imported files, (but didn't finish implementing that because it is not strictly needed yet). I don't think including this comment hurts anything.

Actually [100 characters is the correct line wrapping](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md#indentation-and-line-width) even though the rest of this code tends to use 80 because I prefer 80.

Reply:
OK. fixed to make it 80. I guess it's better to at least be consistent.